Restful api error information

PowderLi opened the pull request:
/kind bug
fix #25564
fix #25629
bring by #24761

PowderLi commented:
/run-cpu-e2e

PowderLi commented:
/run-cpu-e2e

PowderLi commented:
/run-cpu-e2e

PowderLi commented:
/run-cpu-e2e

jaime0815 commented on internal/distributed/proxy/httpserver/handler_v1.go:
```diff
@@ -331,18 +372,35 @@ func (h *Handlers) query(c *gin.Context) {
+		for _, field := range coll.Schema.Fields {
+			if field.DataType == schemapb.DataType_BinaryVector || field.DataType == schemapb.DataType_FloatVector {
+				req.OutputFields = append(req.OutputFields, field.Name)
+				break
```
we'd better append all matched vector fields.

czs007 commented:
/approve

jaime0815 commented on internal/distributed/proxy/httpserver/handler_v1.go:
```diff
@@ -351,49 +409,63 @@ func (h *Handlers) query(c *gin.Context) {
+		req.OutputFields = []string{DefaultOutputFields}
+		for _, field := range coll.Schema.Fields {
+			if field.DataType == schemapb.DataType_BinaryVector || field.DataType == schemapb.DataType_FloatVector {
+				req.OutputFields = append(req.OutputFields, field.Name)
```
we'd better append all matched fields.

PowderLi committed:
Restful api error information
Fix get/query without outputFields return all fields include vector field
Fix check db exists after authorization

Signed-off-by: PowderLi <min.li@zilliz.com>

jaime0815 commented:
/lgtm
